gradle.properties

```diff
@@ -28,18 +28,18 @@ loader_version_range=[4,)
 
 # The unique mod identifier for the mod. Must be lowercase in English locale. Must fit the regex [a-z][a-z0-9_]{1,63}
 # Must match the String constant located in the main mod class annotated with @Mod.
-mod_id=examplemod
+mod_id=transportationmod
 # The human-readable display name for the mod.
-mod_name=Example Mod
+mod_name=first mod
```

Comment:
Why name your mod first mod?

we are working on transportation name it Transportation Mod